clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -1854,6 +1855,9 @@ bool Sema::CheckConstexprFunctionDefinition(const FunctionDecl *NewFD,
     }
   }
 
+  if (Context.getTargetInfo().getTriple().isAArch64())
+    ARM().CheckSMEFunctionDefAttributes(NewFD);
+
```

Comment:
This seems like a weird place to do that (afaict this has nothing to do with constexpr) and I am not sure this code is tested. I suspect `ActOnStartOfLambdaDefinition` - after the `ProcessDeclAttributes` call might be a better place

Reply:
Thanks, `ActOnStartOfLambdaDefinition` does seem like a better place to check the attributes.